onnxruntime/contrib_ops/rocm/bert/attention_impl.h

```diff
@@ -11,6 +11,18 @@
 namespace onnxruntime {
 namespace contrib {
 namespace rocm {
+
+constexpr int kCumulatedSequenceLengthCacheMaxBatchSize = 128;
+
+struct CumulatedSequenceLengthCache {
+  onnxruntime::IAllocatorUniquePtr<void> buffer;
+  int32_t max_batch_size;
+  int32_t sequence_length;
+
+  CumulatedSequenceLengthCache() : max_batch_size(0), sequence_length(0) {}
+  void Initialize(int32_t sequence_length, hipStream_t stream);
```

Comment:
> void Initialize(int32_t sequence_length, hipStream_t stream);

I don't see a definition for this method, nor is it referenced anywhere else. Also, why not have a constructor that takes these parameters and constructs the object correctly?

Reply:
`CumulatedSequenceLengthCache` is some leftover when manually hipifying from CUDA, removed now.